### PR TITLE
fix(docs): fix conflict Fela and FontAwesome

### LINKED ISSUE #950: [Docs] font-family gets randomized in docSite examples
### Steps
After the refactoring of the docSite to be based on Stardust UI, some of our examples are randomly showing Font Awesome 5 Free font instead of default Stardust's Segoe. 

### Expected Result
The same font-family is shown in all the examples.

### Actual Result
The actual result that happened 💣
image
In some examples the FontAwesome 5 Free appears to be set as default.

### Version
master

### Testcase
[Fork, update, and replace this pen to show the bug]:
https://codesandbox.io/s/2l3n74j9y

### PR DESCRIPTION
**Pull request by layershifter**
Fixes #950.

### CODE REVIEW

File: packages/react/src/lib/felaRenderer.tsx [hunk 1]
```diff
@@ -24,6 +31,7 @@ const createRendererConfig = (options: any = {}) => ({
     felaPluginFallbackValue(),
     ...(options.isRtl ? [rtl()] : []),
   ],
+  filterClassName,
```
Comment (layershifter) [inline packages/react/src/lib/felaRenderer.tsx:34]: https://github.com/rofrischmann/fela/blob/v10.2.1/packages/fela/src/generateClassName.js#L25

**Review by miroslavstastny — APPROVED** [review]
Funny :-)

**Review by alinais — APPROVED** [review]

File: packages/react/src/lib/felaRenderer.tsx [hunk 2]
```diff
@@ -7,6 +7,13 @@ import rtl from 'fela-plugin-rtl'
 
 import { Renderer } from '../themes/types'
 
+// Blacklist contains a list of classNames that are used by FontAwesome
+// https://fontawesome.com/how-to-use/on-the-web/referencing-icons/basic-use
+const blacklistedClassNames = ['fa', 'fas', 'far', 'fal', 'fab']
```
Comment (bmdalex) [inline packages/react/src/lib/felaRenderer.tsx:12]: a bit concerned about these as they can change at any time in fontawesome and we have no control over that..

File: packages/react/src/lib/felaRenderer.tsx (hunk 1 above)
Comment (robinweser) [reply packages/react/src/lib/felaRenderer.tsx:34]: Just as I see it here: you can now use the rtl plugin with a theme.direction property to trigger rtl transformation (check the plugin docs) that way you dont have to conditionally add the plugin

Comment (layershifter) [reply packages/react/src/lib/felaRenderer.tsx:34]: Yep, seen these changes. Going to visit this place in a separate PR 👍

File: packages/react/src/lib/felaRenderer.tsx (hunk 2 above)
Comment (kuzhelov) [reply packages/react/src/lib/felaRenderer.tsx:12]: agree - especially given that each theme might load CSS libraries with different set of class names that should be blacklisted. Would expect that this `blacklisted` information will be defined by theme, not (only) general Stardust logic - and, thus, would be extendable

**Pull request merged**